crates/wash-cli/src/app/mod.rs

```diff
@@ -320,8 +320,15 @@ async fn deploy_model(cmd: DeployCommand) -> Result<CommandOutput> {
     let client = connection_opts.into_nats_client().await?;
 
     let app_manifest = match cmd.app_name {
+        Some(source) if source == "-" => load_app_manifest("-".parse()?).await?,
```

Comment:
nit: since we know it's stdin, maybe an enum would prevent a parse/some slightly weird logic here?

Reply:
Yeah I'm not sure it's worth the additional complexity just yet! I think its' still relatively easy to find the code/update this if necessary